### DISCUSSION TITLE
Cloth simulation settings

### DISCUSSION THREAD

**Discussion by wei-ForPrivateUse** [NVIDIA-Omniverse/PhysX#54] [Q&A] [answered]
Thanks for this great work!
The cloth in snippetPBDCloth somehow seems to be too extensible (like jelly? or thin rubber? I think real cloth is way more 'rigid'?）
What can I do to make it look like real cloth (like a coat)? (I read the api document and wiki, but I still don't know how to correct it. :<)

**Comment by DaveGravel** [comment, marked as answer]
You can try with a higher particle iterations.
It costs more process, but it can help.

gParticleSystem->setSolverIterationCounts(10, 1);
The particle system use 4 iterations by default.

You may also get a different result depending on your engine's update loop.
If your physx is updated to (1/60) and you compare with a (1/120) loop you can surely see a difference.
Maybe with the (1/120) loop you don't need to use a higher number of iterations.
Maybe the stiffness and spring values can help too, I just have test the demo quicly.

**Reply by sschirm** [reply]
@DaveGravel description is good. I will just add that lowering the resolution of the cloth will also have a big impact, if that's reasonable for your use-case.
